rust/src/mask/model.rs

```diff
@@ -78,6 +78,8 @@ pub trait IntoPrimitives<P: 'static>: Sized {
     /// to `P`.
     fn into_primitives(self) -> Box<dyn Iterator<Item = Result<P, ModelCastError>>>;
 
+    fn to_primitives(&self) -> Box<dyn Iterator<Item = Result<P, ModelCastError>>>;
```

Comment:
this method should get some documentation since it is in a public trait. to align it with the changes in #435 i would suggest:
```rust
    /// Creates an iterator from numerical values that yields converted primitive values.
    ///
    /// # Errors
    /// Yields an error for each numerical value that can't be converted into a primitive value.
```